docker/README.md

```diff
@@ -0,0 +1,46 @@
+# Caffe standalone Dockerfiles.
+
+The `standalone` subfolder contains docker files for generating both CPU and GPU executable images for Caffe. The images can be built using make, or by running:
+
+```
+docker build -t caffe standalone/gpu
```

Comment:
This is inconsistent with the Makefile.  In the makefile, the name `caffe` (by itself) refers to the CPU version.

I think it would be clearest to tag them with `caffe:gpu` and `caffe:cpu`, or make `caffe` by itself refer to the GPU version since that's the default.